imagery/i.gensig/testsuite/test_keyvalue_result.txt

```diff
@@ -0,0 +1,12 @@
+time=1.103s
```

Comment:
Committed by mistake. `git add {directory}` usually goes wrong. Possibly needs to be added to `.gitignore` and ideally not generated in these cases.

Reply:
I usually was running tests with absolute path and thus test_keyvalue file wasn't an issue :-(

Adding this to the .gitignore is the right thing to do. Could you create a PR?

Reply:
.gitignore done in #2445.